torch/_inductor/fx_passes/ddp_fusion.py

```diff
@@ -95,16 +103,13 @@ def get_comm_block(comm_node: fx.Node) -> Optional[CommBlock]:
     wait_nodes = []
     inputs, _ = tree_flatten((comm_node.args, comm_node.kwargs))
     input_nodes = [inp for inp in inputs if isinstance(inp, fx.Node)]
-    wait_prefixes = "wait_tensor"
+
     # If the users of the wait node are following items, we consinder them
     # to be a part of the output.
     intermediate_outputs = ("split", "reshape", "getitem", "detach", "alias")
 
     first_user = next(iter(comm_node.users))
-    if (
-        len(comm_node.users) == 1
-        and first_user.target == torch.ops._c10d_functional.wait_tensor.default
-    ):
+    if len(comm_node.users) == 1 and is_wait_tensor(first_user):
```

Comment:
Thanks for the update. But we also need to update the call_function below that replace multiple wait_nodes with just one wait_node. This test is green before this PR: `pytest test/distributed/_composable/test_replicate_with_compiler.py  --verbosity=1 -k test_bucketing_concat_op`.